discretize/BaseMesh.py

```diff
@@ -290,11 +351,33 @@ def projectEdgeVector(self, eV):
         ), 'eV must be an ndarray of shape (nE x dim)'
         return np.sum(eV*self.tangents, 1)
 
+    def save(self, filename='mesh.json', verbose=False):
```

Comment:
This might want to be on a base class for simpeg in future (or in the utils of discretize)? It looks like a pretty generic function.

Reply:
Agreed. Once we have injected properties elsewhere in SimPEG we can abstract this up.